Formula/php.rb

```diff
@@ -0,0 +1,333 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net/"
+
+  stable do
+    version "7.1.10"
+    url "http://php.net/get/php-7.1.10.tar.gz/from/this/mirror"
+    sha256 "edc6a7c3fe89419525ce51969c5f48610e53613235bbef255c3a4db33b458083"
+
+    depends_on "libtool" => :run
+    depends_on "mcrypt"
+  end
+
+  devel do
+    url "https://downloads.php.net/~remi/php-7.2.0RC3.tar.gz"
+    sha256 "076491818dd80d723c9232bd4553a95eb30374f3c602d5ca67b95f2ac5e1655a"
+
+    depends_on "argon2"
+    depends_on "libsodium"
+  end
+
+  option "with-thread-safety", "Build with thread safety"
```

Comment:
I don't think we should have any options in this formula.

Reply:
<1% installs PHP with thread safety at the moment so it could probably be removed or that a separate formula is created for it.